wbooks/src/Books.js

```diff
@@ -19,13 +20,9 @@ class Books extends Component {
      return (
        <div className="grid">
         {this.filteredBooks().map(item =>
-          <Link to={"/books/" + item.id} key={item.id}>
+          <Link to={"/books/${item.id}"} key={item.id}>
```

Comment:
You need to use ` instead of " to interpolate strings. give it a quick test just to be sure it works.